pkg/apis/apis/v1alpha1/types.go

```diff
@@ -251,6 +267,26 @@ type APIBindingList struct {
 	Items []APIBinding `json:"items"`
 }
 
+// These are valid conditions of APIExport.
+const (
+	APIExportIdentityValid conditionsv1alpha1.ConditionType = "IdentityValid"
+
+	IdentityVerificationFailedReason = "IdentityVerificationFailed"
+	IdentityGenerationFailedReason   = "IdentityGenerationFailed"
+)
+
+// These are for APIExport identity.
+const (
+	// AnnotationKeyAPIExportCluster is the key for an annotation on an identity secret for an APIExport for the
+	// cluster name for the associated APIExport.
+	AnnotationKeyAPIExportCluster = "apis.kcp.dev/apiexport-cluster"
```

Comment:
why is this needed? Isn't the secret in the same workspace?

Reply:
Probably just me adding unnecessary things. Will see about undoing.

Reply:
Oh yeah, now I remember why I added these. I wanted a way to requeue an APIExport whenever a secret event came in for a secret that was used as an identity. I can probably rework this with an index.